content/blog/posts/devops-daycamp-2015.rst

```diff
@@ -0,0 +1,38 @@
+DevOps DayCamp
+==============
+:date: 2015-10-29
+:author: Amanda Kelner
+:slug: devops-daycamp
```

Comment:
This should be ``devops-daycamp-2015`` as this slug was already used with the previous daycamp post.